spec/serializers/api/v2/shared/import_trade_summary_serializer_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Api::V2::Shared::ImportTradeSummarySerializer do
+  describe '#serializable_hash' do
+    subject(:serializable_hash) { described_class.new(serializable).serializable_hash }
+
+    let(:serializable) do
+      Hashie::TariffMash.new(
```

Comment:
We should try and take a real example of an import trade summary, here, just to validate the integration

Reply:
e.g.

```ruby
let(:serializable) do
  import_measures = [create(:measure, :third_country, :erga_omnes, :with_measure_components)]
  
  described_class.build(import_measures)
end
```